tests/test_pool.py

```diff
@@ -749,9 +777,8 @@ def test_ip_port_n_instance(
 ):
     monkeypatch.delenv("PYMAPDL_START_INSTANCE", raising=False)
     monkeypatch.delenv("PYMAPDL_IP", raising=False)
-    monkeypatch.setenv(
-        "PYMAPDL_MAPDL_EXEC", "/ansys_inc/v222/ansys/bin/ansys222"
-    )  # to avoid trying to find it.
+    monkeypatch.delenv("PYMAPDL_PORT", raising=False)
+    monkeypatch.setenv("PYMAPDL_MAPDL_EXEC", "/ansys_inc/v222/ansys/bin/ansys222")
```

Comment:
Is there a reason why using `v22.2` for this test?

Reply:
It does not matter the version, because MAPDL won't be launched, check the next lines:

```
    with context:
        conf = MapdlPool(
            n_instances=n_instances, ip=ip, port=port, _debug_no_launch=True
        )._debug_no_launch
```

Reply:
Indeed, thanks for the clarification!